e2e/src/usecases/WalletConnectV1.js

```diff
@@ -310,7 +285,7 @@ export default WalletConnect = () => {
     }
 
     let result = walletConnector.sendCustomRequest(customRequest)
-    await waitFor(element(by.text('Sign a Celo transaction')))
+    await waitFor(element(by.text('WalletConnectV1 E2E would like to sign a Celo transaction.')))
```

Comment:
in a future PR i might change these to reference test id's instead :/ it's annoying to remember to update these with copy changes

Reply:
Yes alternatively I wonder if we should load the locale by id and avoid adding testID everywhere. 🤔